internal/http_cache/http_cache.go

```diff
@@ -88,10 +95,13 @@ func checkCacheExists(w http.ResponseWriter, cacheKey string) {
 		TaskIdentification: cirrusTaskIdentification,
 		CacheKey:           cacheKey,
 	}
-	_, err := client.CirrusClient.CacheInfo(context.Background(), &cacheInfoRequest)
+	response, err := client.CirrusClient.CacheInfo(context.Background(), &cacheInfoRequest)
 	if err != nil {
 		w.WriteHeader(http.StatusNotFound)
 	} else {
+		w.Header().Set(CirrusHeaderCreatedBy, strconv.FormatInt(response.Info.CreatedByTaskId, 10))
```

Comment:
The other fields seem to be unused, do we really need them?

Reply:
Can remove. Just exposed all the info available. 🤷

Reply:
See d06f275